supabase/migrations/20260704140000_donor_matching_merge_candidates.sql

```
@@ -0,0 +1,92 @@
+ALTER TABLE public.donors
+  ADD COLUMN IF NOT EXISTS merged_into_donor_id UUID REFERENCES public.donors(id),
+  ADD COLUMN IF NOT EXISTS merged_at TIMESTAMPTZ;
+
+COMMENT ON COLUMN public.donors.merged_into_donor_id IS
+  'Entity resolution (§2.5): when set, this is a MERGED/secondary record redirected to '
+  'the surviving/canonical donor. NULL = canonical record. Never deleted on merge.';
+COMMENT ON COLUMN public.donors.merged_at IS
+  'Timestamp the donor record was merged/redirected into the surviving record.';
+
+CREATE INDEX IF NOT EXISTS donors_merged_into_donor_id_idx
+  ON public.donors (merged_into_donor_id) WHERE merged_into_donor_id IS NOT NULL;
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)